main.py

```diff
@@ -15,6 +15,16 @@ def get_number_from_input(text):
     return int(num)
 
 
+count = 1
+def count_attemps():
```

Comment:
may be this function must be in class ATM. And than I can try to do same things , without using global
and may be it's will be more correctly . idk